core/src/test/java/apoc/spatial/DistanceTest.java

```diff
@@ -57,7 +57,6 @@ public void testCalculateShortestDistanceAndReturnPath() {
     @Test
     public void testSortingPathsOnDistance() {
```

Comment:
There is a commented out test above, is there a todo to uncomment it (if we assume it is the same as ignored tests?). Otherwise, it could also be deleted with this PR :)

Reply:
Good catch, that outcommented test seems to test a proc that does not exist in core anymore, will remove it